docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -652,16 +629,245 @@
+      "text/plain": [
+       "<Figure size 500x500 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_dataset(train_features, train_labels, test_features, test_labels, adhoc_total)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2. Defining the Quantum Kernel\n",
+    "\n",
+    "We proceed with the same kernel setup as it was in the classification task, namely a `ZZFeatureMap` circuit as a feature map and an instance of `FidelityQuantumKernel`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_map = ZZFeatureMap(feature_dimension=2, reps=2, entanglement=\"linear\")\n",
+    "qpca_kernel = FidelityQuantumKernel(fidelity=fidelity, feature_map=feature_map)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, we evaluate kernel matrices for train and test features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "matrix_train = qpca_kernel.evaluate(x_vec=train_features)\n",
+    "matrix_test = qpca_kernel.evaluate(x_vec=test_features, y_vec=test_features)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.3. Comparison of Kernel PCA on gaussian and quantum kernel\n",
+    "\n",
+    "In this section we use the `KernelPCA` implementation from `scikit-learn` with the `kernel` parameter set to \"rbf\" for a gaussian kernel and \"precomputed\" for a quantum kernel. The former is very popular in classical machine learning models, whereas the latter allows using a quantum kernel defined as `qpca_kernel`.\n",
+    "\n",
+    "One can observe that the gaussian kernel based Kernel PCA model fails to make the dataset linearly separable, while the quantum kernel successfully transforms the initial dataset into a linearly separable one.\n",
+    "\n",
+    "While usually PCA is used to reduce the number of features in a dataset, or in other words, such algorithms are used to reduce dimensionality of a dataset, we don't pretend to reduce dimensionality of our dataset here. We keep the number of dimensions and employ kernel PCA mostly for visualization purposes, to show that classification on the transformed dataset becomes easily tractable by linear methods, like logistic regression. We use this method to separate two classes in the principal component space with a `LogisticRegression` model from `scikit-learn`. As usual we train it by calling the `fit` method on the training dataset and evaluate the model for accuracy with `score`."
```

Comment:
```suggestion
    "While usually PCA is used to reduce the number of features in a dataset, or in other words to reduce dimensionality of a dataset, we don't do that here. Rather we keep the number of dimensions and employ the kernel PCA, mostly for visualization purposes, to show that classification on the transformed dataset becomes easily tractable by linear methods, like logistic regression. We use this method to separate two classes in the principal component space with a `LogisticRegression` model from `scikit-learn`. As usual we train it by calling the `fit` method on the training dataset and evaluate the model for accuracy with `score`."
```
I found the first sentence a bit tougher to parse and simplified it a bit. Hope I did not lose any meaning in the process that you meant there.